src/main/scala/com/gyoho/adget/AdgetService.scala

```diff
@@ -0,0 +1,30 @@
+package com.gyoho.adget
+
+import com.gyoho.adget.utils.ScalaHttpClient
+
+object AdgetService {
+  @volatile private var ads: Map[String, Ad] = Map.empty
```

Comment:
Employed `volatile` since we have only one writer, and assignment is a `atomic` operation.
See: https://stackoverflow.com/questions/9749746/what-is-the-difference-between-atomic-volatile-synchronized